mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncloanrepaymenttransacition/SyncLoanRepaymentTransactionFragment.java

```diff
@@ -0,0 +1,276 @@
+import com.mifos.mifosxdroid.adapters.SyncLoanRepaymentAdapter;
+import com.mifos.mifosxdroid.core.MifosBaseActivity;
+import com.mifos.mifosxdroid.core.MifosBaseFragment;
+import com.mifos.mifosxdroid.core.util.Toaster;
+import com.mifos.objects.ErrorSyncServerMessage;
+import com.mifos.objects.PaymentTypeOption;
+import com.mifos.objects.accounts.loan.LoanRepaymentRequest;
+import com.mifos.utils.PrefManager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * Created by Rajan Maurya on 28/07/16.
+ */
+public class SyncLoanRepaymentTransactionFragment extends MifosBaseFragment implements
+        SyncLoanRepaymentTransactionMvpView {
+
+    public final String LOG_TAG = getClass().getSimpleName();
+
+    @BindView(R.id.rv_sync_payload)
+    RecyclerView rv_loan_repayment;
+
+    @BindView(R.id.swipe_container)
+    SwipeRefreshLayout swipeRefreshLayout;
+
+    @BindView(R.id.noPayloadText)
+    TextView mNoPayloadText;
+
+    @BindView(R.id.noPayloadIcon)
+    ImageView mNoPayloadIcon;
+
+    @BindView(R.id.ll_error)
+    LinearLayout ll_error;
+
+    @Inject
+    SyncLoanRepaymentTransactionPresenter mSyncLoanRepaymentTransactionPresenter;
+
+    @Inject
+    SyncLoanRepaymentAdapter mSyncLoanRepaymentAdapter;
+
+    private View rootView;
+
+    private List<LoanRepaymentRequest> mLoanRepaymentRequests;
+
+    private int mClientSyncIndex = 0;
+
+    public static SyncLoanRepaymentTransactionFragment newInstance() {
+        Bundle arguments = new Bundle();
+        SyncLoanRepaymentTransactionFragment fragment = new SyncLoanRepaymentTransactionFragment();
+        fragment.setArguments(arguments);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((MifosBaseActivity) getActivity()).getActivityComponent().inject(this);
+        mLoanRepaymentRequests = new ArrayList<>();
+        setHasOptionsMenu(true);
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        rootView = inflater.inflate(R.layout.fragment_syncpayload, container, false);
+
+        ButterKnife.bind(this, rootView);
+        mSyncLoanRepaymentTransactionPresenter.attachView(this);
+
+        LinearLayoutManager mLayoutManager = new LinearLayoutManager(getActivity());
+        mLayoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        rv_loan_repayment.setLayoutManager(mLayoutManager);
+        rv_loan_repayment.setHasFixedSize(true);
+        rv_loan_repayment.setAdapter(mSyncLoanRepaymentAdapter);
+
+
+        /**
+         * Loading All Loan Repayment Offline Transaction from Database
+         */
+        swipeRefreshLayout.setColorSchemeResources(R.color.blue_light, R.color.green_light, R
```

Comment:
Pass an array instead of typing colors here. 

Like this 
```
<resources>
    <array name="swipeRefreshColors">
        <item>@color/pink</item>
        <item>@color/green</item>
    </array>
</resources>
```

```
swipeRefreshLayout.setColorSchemeColors(getResources().getIntArray(R.array.swipeRefreshColors));
```